pkg/exchange/binance/binanceapi/futures_get_position_risks_request.go

```diff
@@ -8,24 +8,30 @@ import (
 )
 
 type FuturesPositionRisk struct {
-	EntryPrice       fixedpoint.Value           `json:"entryPrice"`
-	MarginType       string                     `json:"marginType"`
-	IsAutoAddMargin  string                     `json:"isAutoAddMargin"`
-	IsolatedMargin   string                     `json:"isolatedMargin"`
-	Leverage         fixedpoint.Value           `json:"leverage"`
-	LiquidationPrice fixedpoint.Value           `json:"liquidationPrice"`
-	MarkPrice        fixedpoint.Value           `json:"markPrice"`
-	MaxNotionalValue fixedpoint.Value           `json:"maxNotionalValue"`
-	PositionAmount   fixedpoint.Value           `json:"positionAmt"`
-	Notional         fixedpoint.Value           `json:"notional"`
-	IsolatedWallet   string                     `json:"isolatedWallet"`
-	Symbol           string                     `json:"symbol"`
-	UnRealizedProfit fixedpoint.Value           `json:"unRealizedProfit"`
-	PositionSide     string                     `json:"positionSide"`
-	UpdateTime       types.MillisecondTimestamp `json:"updateTime"`
+	Symbol                 string           `json:"symbol"`
+	PositionSide           string           `json:"positionSide"`
+	PositionAmount         fixedpoint.Value `json:"positionAmt"`
+	EntryPrice             fixedpoint.Value `json:"entryPrice"`
+	BreakEvenPrice         fixedpoint.Value `json:"breakEvenPrice"`
+	MarkPrice              fixedpoint.Value `json:"markPrice"`
+	UnRealizedProfit       fixedpoint.Value `json:"unRealizedProfit"`
+	LiquidationPrice       fixedpoint.Value `json:"liquidationPrice"`
+	IsolatedMargin         string           `json:"isolatedMargin"`
+	Notional               fixedpoint.Value `json:"notional"`
+	MarginAsset            fixedpoint.Value `json:"marginAsset"`
+	IsolatedWallet         string           `json:"isolatedWallet"`
+	InitialMargin          fixedpoint.Value `json:"initialMargin"`
+	MaintMargin            fixedpoint.Value `json:"maintMargin"`
+	PositionInitialMargin  fixedpoint.Value `json:"positionInitialMargin"`
```

Comment:
@anywhy please don't change the order of the field, it is hard to review the difference